### PR TITLE
Model Patch WeekBasedMaterialDemand v.3.0.1

### PR DESCRIPTION
**Pull request by LukasSchmetz**
## Description

Inconsistency in constraints fixed

Issue: https://github.com/eclipse-tractusx/sldt-semantic-models/issues/777

Closes [#](https://github.com/eclipse-tractusx/sldt-semantic-models/issues/777)

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.6.0)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by LukasSchmetz** [comment]
@jSchuetz88 Kind reminder. Wir wollen im Standard PR auf die jeweils neuen Versionen der Datenmodelle verweisen. Mein letzter Stand war, dass die PRs Anfang der Woche von dir genehmigt werden sollten. Danke.

**Review by jSchuetz88 — CHANGES_REQUESTED** [review]
Thank you for the contribution. Only minor, but mandatory remarks: 
- the prefix as suggested needs to be adjusted to the new version as proposed
- the RELEASE_NOTES needs to be updated accordingly

As this patch does not change the model or anything that would impact the semantic interoperability, and the version 3.0.0 validates against the new version 3.0.1 without any issues (and vice versa), this patch is good to go from my side. 

---

Only one general remark: in a future (major) version, we should discuss the scale of the QuantityFixedPoint (with 1000) and maybe consider a shared model for io.catenax.week_based_material_demand

**Comment by LukasSchmetz** [comment]
@jSchuetz88 Feedback included as requested

**Comment by jSchuetz88** [comment]
Sorry, I thought this Model has already been approved and merged. Looks good from my side.

**Review by jSchuetz88 — APPROVED** [review]
Looks good from my side. All MS II Criteria are passed.

**Comment by LukasSchmetz** [comment]
@jSchuetz88 Thanks for approving. What is now missing / who needs to do what to finish it up?

**Comment by jSchuetz88** [comment]
@agg3fe Could you check?

**Comment by jSchuetz88** [comment]
> @jSchuetz88 Thanks for approving. What is now missing / who needs to do what to finish it up?

Everything is ready. Now we need a committer.

**Comment by jSchuetz88** [comment]
@agg3fe Could you check?

File: io.catenax.week_based_material_demand/RELEASE_NOTES.md
Comment (agg3fe) [inline io.catenax.week_based_material_demand/RELEASE_NOTES.md:5]: should be the date when you got the MS3 approval.

Comment (jSchuetz88) [reply io.catenax.week_based_material_demand/RELEASE_NOTES.md:5]: Sorry, that was my bad as I did not add the label for the approval. I will notify the team.

Comment (jSchuetz88) [inline io.catenax.week_based_material_demand/RELEASE_NOTES.md:5]:
```suggestion
## [3.0.1] - 2024-10-14
```

**Review by agg3fe — APPROVED** [review]

File: io.catenax.week_based_material_demand/RELEASE_NOTES.md
Comment (LukasSchmetz) [reply io.catenax.week_based_material_demand/RELEASE_NOTES.md]: Changed

**Comment by LukasSchmetz** [comment]
@jSchuetz88 Put an empty line. Fine to proceed and finally merge? Thanks

**Pull request merged**
